init.ts

```diff
@@ -99,13 +99,17 @@ async function createSuperuser() {
 export const app = express();
 app.enable('trust proxy');
 
-const doRunTests = (process.env.RUN_TESTS || '').trim() === '1';
 const init = async () => {
 	try {
+		const doRunTests =
+			(process.env.RUN_TESTS || '').trim() === '1'
+				? await import('./test/test-lib/init_mocks')
```

Comment:
I think it's worth renaming that file as it isn't really just initializing mocks anymore (and it'll only diverge more and more over time).

```suggestion
				? await import('./test/test-lib/init_tests')
```